src/Aspire.Cli/Commands/PublishCommand.cs

```diff
@@ -12,23 +12,44 @@
 
 namespace Aspire.Cli.Commands;
 
+internal interface IPublishCommandPrompter
+{
+    Task<string> PromptForPublisherAsync(IEnumerable<string> publishers, CancellationToken cancellationToken);
+}
+
+internal class PublishCommandPrompter(IInteractionService interactionService) : IPublishCommandPrompter
```

Comment:
would a mocking framework help at all with this kind of thing

Reply:
We dont need one.

Reply:
I usually find they are more trouble than they are worth.